internal/cluster/cluster.go

```diff
@@ -764,3 +767,11 @@ func (m Manager) PermanentClustersDeletion(ctx context.Context, olderThen strfmt
 	}
 	return nil
 }
+
+func (m *Manager) GetClusterByKubeKey(key types.NamespacedName) (*common.Cluster, error) {
```

Comment:
Need the same function in `InstallerInternals` that will wrap this one, i don't want to use cluster api in the controller directly.